docs/code/code-repo-management.asciidoc

```diff
@@ -7,17 +7,35 @@ image::images/code-repo-management.png[]
 
 [float]
 ==== Add and delete a repo
-The <<code-import-first-repo, Import your first repository>> page provides step-by-step instructions for adding a GitHub repo to *Code*.  You can fine tune the hostname of the git clone URL in your `kibana.yml` file.
+The <<code-import-first-repo, Import your first repository>> page provides step-by-step instructions for adding a GitHub repo to *Code*.  You can fine-tune the hostname of the git clone URL in your `kibana.yml` file.
 
-For security reasons, Code allows only a few trusted hostnames, such as github.com, by default. You can add an SSH key to {kib} to clone private repos.
+For security reasons, Code allows only a few <<#clone-url-management, trusted hostnames>>, such as github.com. To clone private repositories <<#clone-private-repo-with-ssh-key, add an SSH key>>.
 
-Deleting a repo removes it from local storage and the Elasticsearch index.
+To delete a repository, find the go to the **Repositories** tab, find the name of the repo and click *Delete*.
```

Comment:
Remove "find the"

Also, add comma after repo:  "find the name of the repo, and ..."